libs/core/src/lib/layout-grid/directives/layout-grid-col.directive.ts

```diff
@@ -0,0 +1,88 @@
+import { Directive, ElementRef, Input, OnChanges, OnInit } from '@angular/core';
+import { CSS_CLASS_NAME, GRID_COLUMNS_NUMBER } from '../constants';
+import { applyCssClass } from '../../utils/decorators/apply-css-class.decorator';
+import { CssClassBuilder } from '../../utils/interfaces/css-class-builder.interface';
+
+@Directive({
+    selector: '[fdLayoutGridCol]'
```

Comment:
Could you add also `fd-layout-grid-col` selector, just to keep same convention as on other directives?

Reply:
Done